core/src/main/java/io/paradoxical/cassieq/dataAccess/interfaces/QueueRepository.java

```diff
@@ -47,5 +49,6 @@
 
     void deleteCompletionJob(DeletionJob queue);
 
-    void deleteQueueStats(QueueId id);
-}
\ No newline at end of file
+    void deleteQueueStats(QueueId queueId);
```

Comment:
not sure if this was a merge conflict resolution or not, but pretty sure we removed this.  its not a big deal, i can look later